mscore/exportxml.cpp

```diff
@@ -2150,6 +2181,17 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
                         xml.tagE("stopped");
                         }
                         break;
+                  case ArticulationType::Harmonic:
+/* TODO
+                        {
+                        notations.tag(xml);
+                        technical.tag(xml);
+                        xml.stag("harmonic");
+                        xml.tagE("natural");
+                        xml.etag();
+                        }
+*/
```

Comment:
I believe this should be correct, but I'd rather like @lvinken to cast an eye on this